modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -1,319 +1,458 @@
+The auth domain is `External`, indicating that the user is authenticated on an external server.
+
+NOTE: You can authenticate the external users only using the `PLAIN` method.
+By default, the Couchbase SDK uses PLAIN only when connecting over TLS ports.
+Therefore, external users must connect through TLS ports.
+For more information about the authentication methods, see xref:learn:security/authentication-overview.adoc#authentication-for-applications[Understanding Authentication].
+
+=== Delete a User
+
+To delete a user, do the following:
+
+. In the [.ui]*Users & Groups* tab, select [.ui]*Users*.
+
+. Select the user you want to remove, and then select btn:[Delete].
+The [.ui]*Confirm Delete User* dialog is displayed.
+
+. To confirm the deletion, select btn:[Delete User].
+
+[#lock-and-unlock-user-accounts]
+=== Lock and Unlock User Accounts
+
+IMPORTANT: This is an Enterprise Edition feature.
+
+You can lock user accounts to prevent access to the Couchbase Server, and unlock them to restore access.
+Locking a user account does not delete the user; it only prevents authentication.
+After an account is marked as locked, it prevents the user from logging in or from continuing the existing sessions.
+Locking an account terminates all active UI sessions and long-running connections such as Memcached or streaming connections.
 
 [NOTE]
 ====
-External users can only be authenticated using the `PLAIN` method.
-By default, the Couchbase SDK will only default to PLAIN when connecting over TLS ports, therefore, external users should use the TLS ports.
+* This feature is only available after the entire cluster is upgraded to version 8.0 or later.
 
-More details on authentication methods can be found in xref:learn:security/authentication-overview.adoc#authentication-for-applications[Understanding Authentication]
+* You can lock or unlock only local domain user accounts.
+You cannot lock or unlock external domain user accounts.
 ====
 
+For more information about user account locking, see xref:learn:security/authentication-overview.adoc#blocking-user-authentication[Blocking User Authentication].
+
+To lock or unlock a user account, do the following:
+
+. On the [.ui]*Security* screen, select the [.ui]*Users & Groups* tab, and then select [.ui]*Users*.
+
+. In the [.ui]*Users* list, select the user account you want to lock or unlock.
+
+. Do the following:
+
+** For locking a user account:
+
+.. Select btn:[Lock].
+The [.ui]*Confirm Lock User* dialog is displayed.
+
+.. Select btn:[Lock User] to confirm and save the changes.
+
+** For unlocking a user account:
+
+.. Select btn:[Unlock].
+The [.ui]*Confirm Unlock User* dialog is displayed.
+
+.. Select btn:[Unlock User] to confirm and save the changes.
+
+When a user attempts to log in to their account after it's locked, they see the following error message: 
+
+----
+Login failed. Please try again.
```

Comment:
Mmmm. I would have said `Account Locked` so they know what has happened. Just my opinion, however.

Reply:
The context from engineering side is that we were designing this alongside bruteforce mitigations that would have potentially locked users after too many auth attempts.  If we returned that the account was locked whether or not the password was right, then that would be leaking the existence of users, and if we returned that the account was locked only when the password was right, then that would provide a bypass for the bruteforce mitigation as the attacker could just keep trying until they get the "locked" error. Because we want to leave open this possible functionality in future (we didn't manage to settle on an appropriate solution in time for 8.0), we had to use the generic login failed error for locked users.